byron/ledger/executable-spec/test/Ledger/UTxO/Properties.hs

```diff
@@ -90,3 +95,54 @@ tracesAreClassified = withTests 200 . property $ do
     classify "avg nr. outputs [25, 100)" $ 25 <= avgOutputs && avgOutputs < 100
     classify ">= 100"                    $ 100 <= avgOutputs
   success
+
+relevantCasesAreCovered :: Property
+relevantCasesAreCovered = withTests 400 $ property $ do
+  let tl = 100
+  tr <- forAll (trace @UTXOW tl)
+  let n :: Integer
+      n = fromIntegral $ traceLength tr
+
+  when (n > 0) $ do
+    let
+        ss = preStatesAndSignals OldestFirst tr
+        txs = (body . view _2) <$> ss
+
+        nrInputs, nrOutputs :: Double
```

Comment:
Can't we re-use these from the previous function? (I'd just take these our of the `where` clause ...

Reply:
done, tx